src/versionbitsinfo.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2016-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <versionbitsinfo.h>
+
+#include <consensus/params.h>
+
+const struct VBDeploymentInfo VersionBitsDeploymentInfo[Consensus::MAX_VERSION_BITS_DEPLOYMENTS] = {
+    {
+        /*.name =*/ "testdummy",
+        /*.gbt_force =*/ true,
+    },
+    {
+        /*.name =*/ "csv",
+        /*.gbt_force =*/ true,
+    },
+    {
+        /*.name =*/ "segwit",
+        /*.gbt_force =*/ true,
+    }
```

Comment:
```
{
    /*.name =*/ "nversionbips",
    /*.gbt_force =*/ true,
}
```
Is missing from here. See this file in 0.14.0: https://github.com/vertcoin-project/vertcoin-core/blob/0.14.0/src/versionbits.cpp#L21. This breaks `getblockchaininfo` among other things.